src/QuantumInfo.jl

```diff
@@ -1,8 +1,13 @@
-VERSION >= v"0.4.0-dev+6521" && __precompile__()
 module QuantumInfo
-  using Cliffords
+using Cliffords
+import LinearAlgebra
 
-  include("basics.jl")
-  include("open-systems.jl")
+# As a stopgap, reintroduce the old `eye`.
```

Comment:
+1

Reply:
There is an `Eye` in [FillArrays.jl](https://github.com/JuliaArrays/FillArrays.jl) that potentially could make many calculations, such as `kron`, much more efficient. But most of the methods to take advantage of the structure have not been written. Same goes for the types `Diagonal` and `Fill` (lazy `Array`s filled with one value). I wrote several of the methods, including `kron`, and collected them in [IdentityMatrix.jl](https://github.com/jlapeyre/IdentityMatrix.jl). They should act roughly like drop-in replacements, overriding default methods.  Turning them into PRs to `LinearAlgebra` and `FillArrays` would take a far more time than writing them of course. But, I or someone will do it sooner or later. In the mean time,  if memory or cpu efficiency becomes a problem, you may want to try the package.